nacl/variables.tf

```diff
@@ -0,0 +1,9 @@
+variable "vpc_id" {}
+
+variable "common_tags" {}
+
+variable "ingress_rules" {}
+
+variable "name" {}
```

Comment:
Maybe either make this a more descriptive name, or put description against this variable?

Reply:
I quite like the short names so I've put a description in.